### PR TITLE
AdminGeoFinder implements FromIterator

### PR DESCRIPTION
**Pull request by datanel**
fix https://github.com/CanalTP/mimirsbrunn/issues/96

thanks to @TeXitoi

### CODE REVIEW

File: src/admin_geofinder.rs [hunk 1]
```diff
@@ -86,6 +87,19 @@ impl AdminGeoFinder {
...
+    fn from_iter<I: IntoIterator<Item=Rc<Admin>>>(admins: I) -> Self {
+        let mut geofinder = AdminGeoFinder::new();
+
+        for admin in admins {
+            geofinder.insert(admin.clone());
+        }
+
+        geofinder
+    }
+}
```
Comment (TeXitoi) [inline src/admin_geofinder.rs]: Even better using Into: https://is.gd/pykDBx

File: src/bin/osm2mimir.rs [hunk 2]
```diff
@@ -205,12 +205,7 @@ fn administrative_regions(pbf: &mut OsmPbfReader, levels: BTreeSet<u32>) -> Admi
...
 
 fn make_admin_geofinder(admins: &AdminsVec) -> AdminGeoFinder {
-    let mut geofinder = AdminGeoFinder::new();
-
-    for a in admins {
-        geofinder.insert(a.clone());
-    }
-    geofinder
+    admins.iter().cloned().collect()
 }
```
Comment (TeXitoi) [inline src/bin/osm2mimir.rs]: This function become quite useless, no?

Comment (datanel) [inline src/bin/osm2mimir.rs]: absolutely

File: src/admin_geofinder.rs (hunk 1 above)
Comment (datanel) [inline src/admin_geofinder.rs]: done

**Pull request merged**